app/assets/javascripts/components/client_comments.jsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * @prop comments - comments associated with current client
+ * @prop client - current client
+ * @prop user - current user
+ */
+
+class ClientComments extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      comments: []
+    };
+  }
+
+  componentDidMount() {
+		Requester.get(`/api/comments/client/${this.props.client.id}`).then((comments) => {
+			this.setState({ comments });
+		});
+	}
+
+  addComment(val) {
+    const newComment = {content: val,
+                        client_id: this.props.client.id,
+                        thread_id: 1,
+                        user_id: this.props.user.id
+                        }
+    this.state.comments.push(newComment);
+    this.setState({ comments: this.state.comments });
+    Requester.post(`/api/comments`, newComment)
```

Comment:
We prob want to make the network request first and see if it succeeded before we do anything to the client

Reply:
^ you can use the .then callback of the post to setState if the post succeeded, and do nothing if it fails (or render an error message in the component somewhere)